UndertaleModLib/UndertaleLists.cs

```diff
@@ -7,171 +7,214 @@
 
 namespace UndertaleModLib
 {
-    public abstract class UndertaleListBase<T> : ObservableCollection<T>, UndertaleObject
+    /// <summary>
+    /// Basic observable list type, usable for data models to be presented in interfaces.
+    /// </summary>
+    public class UndertaleObservableList<T> : ObservableCollection<T>
```

Comment:
for your future data models, are those intended to have their own serialize and unserialize functions as well?

Reply:
Was more thinking on whether we want to enforce a serialization and deserialization method, a la interface
But it sounds like for that, its not the physicsvertices that would be able to implement it, but rather the (de)serialization of undertalegameobject, so it doesnt work nicely?

Reply:
Yeah, this is deliberately *not* inheriting from `UndertaleObject` (the interface with `Serialize()`, etc.). Just meant for a list that can be added to efficiently during deserialization, without having any purview over how it's (de)serialized. (And then to be observable for the GUI as well.)